recipe/meta.yaml

```diff
@@ -21,12 +21,12 @@ requirements:
     - ninja      # [win]
 
```

Comment:
I think you don't need ``clang``

Reply:
Thanks for looking into this. I needthe compiler under windows to be clang-cl. I thought that this is the way to achieve it?

Reply:
dont you need to skip ``{{ compiler('c') }}`` and ``{{ compiler('cxx') }}`` for windows then?

Reply:
ok will do that, thanks.